src/codemagic/tools/app_store_connect/action_groups/profiles_action_group.py

```diff
@@ -0,0 +1,147 @@
+        "create",
+        BundleIdArgument.BUNDLE_ID_RESOURCE_ID,
+        CertificateArgument.CERTIFICATE_RESOURCE_IDS,
+        ProfileArgument.DEVICE_RESOURCE_IDS,
+        ProfileArgument.PROFILE_TYPE,
+        ProfileArgument.PROFILE_NAME,
+        CommonArgument.SAVE,
+        action_group=AppStoreConnectActionGroup.PROFILES,
+        deprecation_info=cli.ActionDeprecationInfo("create-profile", "0.49.0"),
+    )
+    def create_profile(
+        self,
+        bundle_id_resource_id: ResourceId,
+        certificate_resource_ids: Sequence[ResourceId],
+        device_resource_ids: Optional[Sequence[ResourceId]] = None,
+        profile_type: ProfileType = ProfileType.IOS_APP_DEVELOPMENT,
+        profile_name: Optional[str] = None,
+        save: bool = False,
+        should_print: bool = True,
+    ) -> Profile:
+        """
+        Create provisioning profile of given type
+        """
+
+        bundle_id = self.get_bundle_id(bundle_id_resource_id, should_print=False)
+        if profile_name:
+            name = profile_name
+        elif profile_name is None:
+            name = f"{bundle_id.attributes.name} {profile_type.value.lower()} {int(time.time())}"
+        else:
+            raise AppStoreConnectError(f'"{profile_name}" is not a valid {Profile} name')
+
+        create_params = dict(
+            name=name,
+            profile_type=profile_type,
+            bundle_id=bundle_id_resource_id,
+            certificates=certificate_resource_ids,
+            devices=[],
+            omit_keys=["devices"],
+        )
+        if profile_type.devices_required():
+            create_params["devices"] = list(device_resource_ids) if device_resource_ids else []
+
+        try:
+            profile = self._create_resource(self.api_client.profiles, should_print, **create_params)
+        except ValueError as ve:
+            raise AppStoreConnectError(str(ve)) from ve
+
+        if save:
+            self._save_profile(profile)
+        return profile
+
+    @cli.action(
+        "get",
+        ProfileArgument.PROFILE_RESOURCE_ID,
+        CommonArgument.SAVE,
+        action_group=AppStoreConnectActionGroup.PROFILES,
+        deprecation_info=cli.ActionDeprecationInfo("get-profile", "0.49.0"),
+    )
+    def get_profile(
+        self,
+        profile_resource_id: ResourceId,
+        save: bool = False,
+        should_print: bool = True,
+    ) -> Profile:
+        """
+        Get specified Profile from Apple Developer portal
+        """
+
+        profile = self._get_resource(profile_resource_id, self.api_client.profiles, should_print)
+        if save:
+            self._save_profile(profile)
+        return profile
+
+    @cli.action(
+        "delete",
+        ProfileArgument.PROFILE_RESOURCE_ID,
+        CommonArgument.IGNORE_NOT_FOUND,
+        action_group=AppStoreConnectActionGroup.PROFILES,
+        deprecation_info=cli.ActionDeprecationInfo("delete-profile", "0.49.0"),
+    )
+    def delete_profile(
```

Comment:
Used to be defined in https://github.com/codemagic-ci-cd/cli-tools/blob/3670fa624997f17cb437bc88198f7bdee3594f35/src/codemagic/tools/app_store_connect.py#L737-L751